client/src/components/QA/QaIndex.jsx

```diff
@@ -1463,7 +1463,7 @@ const QaIndex = () => {
       <h3>QUESTIONS & ANSWERS</h3>
       <QaSearchBar />
       <QaQuestionsList allQuestions={allQuestions} qLimit={qLimit} questions={questions}/>
-      <QaLoadAdd qLimitPlusTwo={qLimitPlusTwo}/>
+      <QaLoadAdd qLimitPlusTwo={qLimitPlusTwo}  product_id={allQuestions.product_id}/>
```

Comment:
If you can try changing the product_id prop to camel case (unless that's how it is in the data set)